Reference_Manual.md

```diff
@@ -1300,12 +1299,12 @@ You can check `BasicTLSClient.cpp` example for a full featured example.
     auto hz = hazelcast::new_client(std::move(config)).get();
 ```
 
-With the above configuration, both the client and the server authenticate each other mutually. The client validates server certificate and the server validates the client with the client certificate.
+With the above configuration, both the client and server authenticate each other mutually. The client validates the server certificate and the server validates the client with the client certificate.
 
-You can check `mutual_authentication.cpp` example for a full featured example.
+You can check `mutual_authentication.cpp` for a full featured example.
 
 #### 6.1.2.3. Constraining The Used Cipher List
-In some cases, you may want to limit the cipher suites allowed for a client to while communicating the server. This can also be configured using the `ssl_config`. Here is an example configuration to enable cipher list which only allow "HIGH" ciphers.
+In some cases, you may want to limit the cipher suites allowed for a client while communicating with the server. This can also be configured using `ssl_config`. Here is an example configuration to enable cipher list which only allow only "HIGH" ciphers.
```

Comment:
```suggestion
In some cases, you may want to limit the cipher suites allowed for a client while communicating with the server. This can also be configured using `ssl_config`. Here is an example configuration to enable cipher list which allow only "HIGH" ciphers.
```